stage1/init/init.go

```diff
@@ -381,6 +375,20 @@ func stage1() int {
 		return 3
 	}
 
+	// write ppid file as specified in
+	// Documentation/devel/stage1-implementors-guide.md
+	out, err := os.Getwd()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Cannot get current working directory: %v\n", err)
+		return 4
+	}
+	err = ioutil.WriteFile(filepath.Join(out, "ppid"),
```

Comment:
Even though you describe it in "Documentation/devel/stage1-implementors-guide.md", it's confusing seeing "ppid" and then `os.Getpid()`, maybe add a comment like

"we are the parent of the process that is PID 1 in the container so we write our PID to "ppid""